meson.build

```diff
@@ -1316,6 +1330,19 @@ else
 endif
 conf.set10('ENABLE_IMPORTD', have)
 
+want_homed = get_option('homed')
+if want_homed != 'false'
```

Comment:
This doesn't work because the option is a boolean. I'll submit a patch for this after merging.